### PR TITLE
Moved uncaughtExceptions handler to cli/errors

### PR DESCRIPTION
**Pull request by SwinX**

### CODE REVIEW

File: lib/cli/errors.js [hunk 1]
```diff
@@ -18,6 +18,13 @@ exports.handleErrors = function(error) {
     return 1;
 };
 
+exports.handleUncaughtExceptions = function() {
+    process.on('uncaughtException', function(err) {
+        console.error(err);
```
Comment (SevInf) [inline lib/cli/errors.js]: I would use `process.exit(handleErrors(error))` to get an output and exit code consistent with the rest of the error reports.

**Comment by SwinX** [comment]
:up:

**Comment by SevInf** [comment]
:ok:

**Pull request merged**